packages/tables/src/Commands/MakeTableCommand.php

```diff
@@ -24,11 +26,13 @@ class MakeTableCommand extends Command
 
     public function handle(): int
     {
-        $component = (string) str($this->argument('name') ?? text(
-            label: 'What is the table name?',
-            placeholder: 'Products/ListProducts',
-            required: true,
-        ))
+        $component = (string) str(
```

Comment:
Please remove all formatting changes in this file and also MakeWidgetCommand

Reply:
Could this formatting be as a result of `composer cs` command or maybe from my IDE formatting.